onnx/inliner/inliner.cc

```diff
@@ -720,5 +722,60 @@ void InlineSelectedFunctions(
   }
   InlinerImpl::InlineSelectedFunctions(model, to_inline, schema_registry);
 }
+
+// Implementation of the Renamer class using InliningRenamer for proper graph-value attribute handling
+class Renamer::Impl : public InliningRenamer {
+ private:
+  std::unique_ptr<NameGenerator> generator_;
+
+ public:
+  Impl(const std::string& prefix, const GraphProto& graph)
+      : InliningRenamer("__" + prefix, *(generator_ = std::make_unique<NameGenerator>(graph))) {}
```

Comment:
The test-case is crashing in this line. Can you simplify the implementation by replacing  inheritance of InliningRenamer by delegation? Make the NameGenerator an instance of this class, and add an instance of InliningRenamer (using that NameGenerator). Avoid the unique pointer.